### ISSUE TITLE
Move to HTTPS?

### ISSUE THREAD

**Issue by tomekwi** [jgorset/git.io#3]
`git.io` (the program) stopped working for me recently:

```sh
$  git.io --code=elm-platform https://github.com/elm-lang/elm-platform
https://git.io/

$ curl 'https://git.io/elm-platform'
No url found for elm-platform
```

I guess it might be caused by http://git.io moving to https://git.io:

```sh
$ curl -i http://git.io -F "url=https://github.com/elm-lang/elm-platform" -F "code=elm-platform"
HTTP/1.1 100 Continue

HTTP/1.1 301 Moved Permanently
Server: Cowboy
Connection: close
Date: Sat, 13 Feb 2016 12:27:02 GMT
Status: 301 Moved Permanently
Content-Type: text/html
Location: https://git.io/
Via: 1.1 vegur

<html><body>You are being <a href="https://git.io/">redirected</a>.</body></html>



$ curl -i https://git.io -F "url=https://github.com/elm-lang/elm-platform" -F "code=elm-platform"
HTTP/1.1 100 Continue

HTTP/1.1 201 Created
Server: Cowboy
Connection: keep-alive
Date: Sat, 13 Feb 2016 12:27:19 GMT
Status: 201 Created
Content-Type: text/html;charset=utf-8
Location: https://git.io/elm-platform
Content-Length: 40
X-Xss-Protection: 1; mode=block
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Runtime: 0.186326
X-Node: 5ecf37e0-a6ed-482b-be65-483d60b87255
X-Revision: 4fdc60de6311e6a3aa31e19bc7b3aad7e85d33a6
Strict-Transport-Security: max-age=31536000; includeSubDomains
Via: 1.1 vegur

https://github.com/elm-lang/elm-platform
```

Hope it helps!

**Comment by Parad0X** [comment]
https://github.com/jgorset/git.io/pull/4

**Comment by jgorset** [comment]
Thanks @tomekwi and @Parad0X!

**Comment by jgorset** [comment]
I merged #4.
